container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/kubernetes_kurtosis_backend.go

```diff
@@ -487,6 +489,11 @@ func (backend *KubernetesKurtosisBackend) BuildImage(ctx context.Context, imageN
 	return "", stacktrace.NewError("Building images isn't yet implemented in Kubernetes.")
 }
 
+func (backend *KubernetesKurtosisBackend) NixBuild(ctx context.Context, nixBuildSpec *nix_build_spec.NixBuildSpec) (string, error) {
```

Comment:
Could you add this diff on this doc https://www.notion.so/kurtosistech/Docker-Kubernetes-Inconsistencies-Discrepancies-ff62b3970c5e44fab4984635459fc825 ?

Reply:
Added!